examples/health-status/src/main/java/io/littlehorse/examples/HealthStatus.java

```diff
@@ -0,0 +1,93 @@
+package io.littlehorse.examples;
+
+import io.littlehorse.sdk.common.config.LHConfig;
+import io.littlehorse.sdk.common.proto.VariableType;
+import io.littlehorse.sdk.wfsdk.WfRunVariable;
+import io.littlehorse.sdk.wfsdk.Workflow;
+import io.littlehorse.sdk.wfsdk.internal.WorkflowImpl;
+import io.littlehorse.sdk.worker.LHTaskWorker;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Properties;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+/*
+ * This is a simple example, which does
+ * 1. Declare an "input-name" variable of type String
+ * 2. Pass that variable into the execution of the "greet" task.
+ * 3. Shows the cluster's health status.
+ */
+public class HealthStatus {
+
+
+    private static final Logger log = LoggerFactory.getLogger(HealthStatus.class);
+
+    public static Workflow getWorkflow() {
+        return new WorkflowImpl(
+            "example-health-status",
+            wf -> {
+                WfRunVariable theName = wf.addVariable("input-name", VariableType.STR).searchable();
```

Comment:
How is this any different from the `BasicExample`?

Reply:
i will delete the examples